### PR TITLE
fix: Respect contracts directory defined in adonisjrc.json

### PR DESCRIPTION
**Pull request by TiBianMod**
The `node ace configure @adonisjs/mail` ignores the defined `contracts` directory from the user inside `.adonisjrc.json`

This PR will respect the `contracts` directory defined by the user inside `.adonisjrc.json` and will fallback to default if not present.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/adonisjs/mail/blob/master/.github/CONTRIBUTING.md) doc
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation (if appropriate)

## Further comments

@thetutlage point me to right direction for adding tests if necessary.
I can't find related tests for `node ace configure @adonisjs/mail`

### CODE REVIEW

**Comment by TiBianMod** [comment]
ping @thetutlage

File: instructions.ts [hunk 1]
```diff
@@ -126,7 +126,8 @@ export default async function instructions(
   /**
    * Create the mail contracts file
    */
-  const contractsPath = app.makePath('contracts/mail.ts')
+  const contractsDir = app.directoriesMap.get('contracts') || 'contracts'
```
Comment (RomainLanz) [inline instructions.ts:129]: Can we use `path.join` instead?

```suggestion
  const contractsDirectory = app.directoriesMap.get('contracts') || 'contracts'
  const contractPath = join(contractsDirectory, 'mail.ts')
```

File: instructions.ts [hunk 2]
```diff
@@ -141,7 +142,7 @@ export default async function instructions(
       sparkpost: mailDrivers.includes('sparkpost'),
     })
     .commit()
-  sink.logger.action('create').succeeded('contracts/mail.ts')
+  sink.logger.action('create').succeeded(`${contractsDir}/mail.ts`)
```
Comment (RomainLanz) [inline instructions.ts:145]:
```suggestion
  sink.logger.action('create').succeeded(contractPath)
```

**Review by RomainLanz — CHANGES_REQUESTED** [review]

**Comment by TiBianMod** [comment]
I have reverted the changes back to initial PR.
Let's keep consistency with the line [123-124](https://github.com/TiBianMod/adonisjs-mail/blob/develop/instructions.ts#L123)

** Also in your commit was an error on [line 131](https://github.com/adonisjs/mail/pull/81/commits/7c5b232de4ea6c0570a13a47706652333aaa1835) :-(

**Comment by thetutlage** [comment]
@TiBianMod Thanks for the PR and sorry for taking that long :)

**Comment by TiBianMod** [comment]
@thetutlage no problem mate 😄

**Pull request merged**